packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -12,9 +14,21 @@ const TIMEOUT = 20000
 
 describe('Payment', function () {
   let testContext: XrplIntegrationTestContext
+  let payment_txn_example: Payment
+  let amount_: string
```

Comment:
```suggestion
  let paymentTx: Payment
  let amount: string
```

Using `_` is Pythonic. We want to use JS convention.

Reply:
updated in [c49957c](https://github.com/XRPLF/xrpl.js/pull/2689/commits/c49957c13fdc6eee0393057a662500d8af08084f)